### PR TITLE
Add graders for QE badge 5

### PR DESCRIPTION
**Pull request by quantum-kittens**

### CODE REVIEW

File: qc_grader/challenges/quantum_explorers23/__init__.py [hunk 1]
```diff
@@ -25,3 +25,24 @@
...
+    grade_badge5_ex6,
+    grade_badge5_ex7,
+    grade_badge5_ex8,
+    grade_badge5_ex9,
+    grade_badge5_ex10,
+    grade_badge5_ex11,
+    grade_badge5_ex12,
+    grade_badge5_ex13,
+    grade_badge5_ex14,
+    grade_badge5_ex15,
```
Comment (vabarbosa) [inline qc_grader/challenges/quantum_explorers23/__init__.py:45]: these two (`grade_badge5_ex14` & `grade_badge5_ex15`) do not exist in the `badge_5.py` file

File: qc_grader/challenges/quantum_explorers23/badge_5.py [hunk 2]
```diff
@@ -0,0 +1,167 @@
...
+        _challenge_id, 
+        return_response=True
+    )
+    print(message)
+
+
+@typechecked
+def grade_badge5_code(qsvc: qiskit_machine_learning.algorithms.classifiers.qsvc.QSVC) -> None:
+    status, _, message = grade(
+        circuit,
```
Comment (vabarbosa) [inline qc_grader/challenges/quantum_explorers23/badge_5.py:151]: `circuit` does not exist here. did you mean `qsvc`?

**Review by vabarbosa — CHANGES_REQUESTED** [review]
thanks. just have a couple comments (see below)

File: qc_grader/challenges/quantum_explorers23/__init__.py (hunk 1 above)
Comment (quantum-kittens) [reply qc_grader/challenges/quantum_explorers23/__init__.py:45]: @vabarbosa yes, thank you. added the missing graders to badge5.py

**Pull request merged**